src-2.0/components/ActionBar.tsx

```
@@ -0,0 +1,113 @@
+
+  return (
+    <div className={styles.actionBar}>
+      <div className={styles.actionBarLeft}>
+        <span className={styles.actionBarInfo}>
+          {currentCollectionName}
+        </span>
+      </div>
+      <div className={styles.actionBarRight}>
+        <a
+          href="https://github.com/mnttnm/figma-variable-explorer"
+          target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)